core/common/src/test/java/alluxio/uri/AuthorityTest.java

```diff
@@ -24,9 +24,10 @@
   @Test
   public void authorityFromStringTest() {
     assertTrue(Authority.fromString("localhost:19998") instanceof SingleMasterAuthority);
+    assertTrue(Authority.fromString("127.0.0.1:19998") instanceof SingleMasterAuthority);
```

Comment:
Is this what failed before this fix?

Reply:
This will be passed before the fix, I just add more cases of hostname here.